vmb/matrimony/models/profiles.py

```diff
@@ -634,34 +636,6 @@ def generate_profile_id(self):
         )
         return f"{settings.PROFILE_ID_PREFIX}{digest}"
 
-    def update_stats(self):
-        _, created = MatrimonyProfileStats.objects.get_or_create(profile=self)
```

Comment:
There's no need to move the update stats method from here. You will just need to call it from the signal handler.

Reply:
@bhawanabadlani I had asked not to move this code from here, but call this code.